hbase-resource-bundle/src/main/resources/META-INF/LICENSE.vm

```diff
@@ -1469,9 +1469,12 @@ ${dep.scm.url}
   "Apache License, Version 2.0" is the official name in the past
   "The Apache Software License, Version 2.0" is the official name in the past too
   "Apache Software License - Version 2.0" is what jetty uses in pom
+  "Apache License, version 2.0" and "Apache License 2.0" are used by Jboss artifacts
   "Apache-2.0" is the new short official name
 *#
 #elseif( !(${dep.licenses[0].name.contains("Apache License, Version 2.0")}
+  || ${dep.licenses[0].name.contains("Apache License, version 2.0")}
```

Comment:
If the license check fails because the exact text doesn't match, we typically override the dependency license by updating this file: hbase-resource-bundle/src/main/resources/supplemental-models.xml